packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -1740,4 +1663,226 @@ describe("Garbage Collection Tests", () => {
 		assert.strictEqual(garbageCollector.tombstones.length, 0, "Expecting 0 tombstone nodes");
 		assert.strictEqual(garbageCollector.deletedNodes.size, 0, "Expecting 0 deleted nodes");
 	});
+
+	describe("GC stats", () => {
+		let garbageCollector: IGarbageCollector;
+		let initialStats: IGCStats;
+
+		const tests = (sweepEnabled: boolean) => {
+			beforeEach(() => {
+				// Set up initial GC graph with 5 nodes and 2 are unreferenced.
+				defaultGCData.gcNodes["/"] = [nodes[0]];
+				defaultGCData.gcNodes[nodes[0]] = [nodes[1]];
+				defaultGCData.gcNodes[nodes[1]] = [];
+				defaultGCData.gcNodes[nodes[2]] = [];
+				defaultGCData.gcNodes[nodes[3]] = [];
+
+				// Set up the initial GC stats based on the initial GC graph.
```

Comment:
Super Nit: I'd call this expectedInitialStats then you can get rid of the `expectedStats` variable in each test